LaunchAtLogin/LaunchAtLogin.swift

```diff
@@ -1,21 +1,147 @@
+import Combine
 import Foundation
 import ServiceManagement
 
-public struct LaunchAtLogin {
-	private static let id = "\(Bundle.main.bundleIdentifier!)-LaunchAtLoginHelper"
+public enum LaunchAtLogin {
+    public static let kvo = LaunchAtLoginKVO()
 
-	public static var isEnabled: Bool {
-		get {
-			guard let jobs = (SMCopyAllJobDictionaries(kSMDomainUserLaunchd).takeRetainedValue() as? [[String: AnyObject]]) else {
-				return false
-			}
+    @available(macOS 10.15, *)
+    public static let observable = LaunchAtLoginObservable()
 
-			let job = jobs.first { $0["Label"] as! String == id }
+    @available(macOS 10.15, *)
+    private static var _publisher = CurrentValueSubject<Bool, Never>(isEnabled)
+    @available(macOS 10.15, *)
+    public static var publisher = _publisher.eraseToAnyPublisher()
 
-			return job?["OnDemand"] as? Bool ?? false
-		}
-		set {
-			SMLoginItemSetEnabled(id as CFString, newValue)
-		}
-	}
+    private static let id = "\(Bundle.main.bundleIdentifier!)-LaunchAtLoginHelper"
+
+    public static var isEnabled: Bool {
+        get {
+            guard let jobs = (SMCopyAllJobDictionaries(kSMDomainUserLaunchd).takeRetainedValue() as? [[String: AnyObject]]) else {
+                return false
+            }
+
+            let job = jobs.first { $0["Label"] as! String == id }
+
+            return job?["OnDemand"] as? Bool ?? false
+        }
+        set {
+            if #available(macOS 10.15, *) {
+                observable.objectWillChange.send()
+            }
+
+            kvo.willChangeValue(for: \.isEnabled)
+            SMLoginItemSetEnabled(id as CFString, newValue)
+            kvo.didChangeValue(for: \.isEnabled)
+
+            if #available(macOS 10.15, *) {
+                _publisher.send(newValue)
+            }
+        }
+    }
+}
+
+// MARK: - LaunchAtLoginObservable
+extension LaunchAtLogin {
+    @available(macOS 10.15, *)
+    public final class LaunchAtLoginObservable: ObservableObject {
```

Comment:
```suggestion
    public final class Observable: ObservableObject {
```

since it's already namespaced. Same with the others.